precompiles/bech32/Bech32.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.17;
```

Comment:
is this necessary?

Reply:
Still best practice to use newest version of Solidity so it enforces devs to use it in their contracts as well to avoid known issues with older versions of Solidity.